client/components/happychat/composer.jsx

```diff
@@ -11,7 +11,7 @@ import { isEmpty } from 'lodash';
 /**
  * Internal dependencies
  */
-import { sendChatMessage, setChatMessage } from 'state/happychat/actions';
+import { sendChatMessage, setChatMessage } from 'state/happychat/connection/actions';
```

Comment:
I think this will have a conflict with https://github.com/Automattic/wp-calypso/pull/18743/files#diff-08015260354feffd637fba80b7826c58R15

Reply:
Good catch! `setChatMessage` is one action I'm unsure about. The other actions mimic pretty well the SocketIO events, so I see them more in the happychat.connection side of things. One follow-up I want to do after these PRs are merged is to rename the actions, so, hopefully, the meaning and boundaries are clearer.

`setChatMessage` confuses me, though: it stores the current chat message and sends the `typing` SocketIO event. Do you think it would make sense to have separate `sendTyping` and `setChatMessage` actions? That's something I could address in an upcoming PR if we feel is worth the separation.

Regarding your question: I don't have strong preferences, but I I lean towards packaging `setChatMessage` within the happychat.chat PR and leaving the other ones in happychat.connection.

Reply:
> Do you think it would make sense to have separate sendTyping and setChatMessage actions? 

I guess the separation would be nice, if it's not too much trouble.

> Regarding your question: I don't have strong preferences, but I I lean towards packaging setChatMessage within the happychat.chat PR.
`happychat.chat` makes more sense for me too